Source/Widgets/CesiumWidget/CesiumWidget.js

```diff
@@ -294,7 +295,7 @@ define([
             }
 
             //Set the base imagery layer
-            var imageryProvider = options.imageryProvider;
+            var imageryProvider = (options.globe === false) ? false : options.imageryProvider;
```

Comment:
Similar checks probably need to be added for the properties I mentioned in my previous comment.